otx/cli/manager/config_manager.py

```diff
@@ -115,6 +117,20 @@ def data_config_file_path(self) -> Path:
             raise FileNotFoundError(f"Not found: {self.args.data}")
         return self.workspace_root / "data.yaml"
 
+    @property
+    def output_path(self) -> Path:
+        """The path of output directory for workspace.
+
+        Returns:
+            Path: Path of output directory.
+        """
+        if "output" in self.args and self.args.output:
+            output_path = Path(self.args.output)
+        else:
+            output_path = self.workspace_root / "outputs" / self.create_date
+        output_path.mkdir(exist_ok=True, parents=True)
```

Comment:
```suggestion
        if not output_path.exists():
            output_path.mkdir(exist_ok=True, parents=True)
```
To consider a case where the function is called multiple times, how about changing code like this?

Reply:
I fixed this. https://github.com/openvinotoolkit/training_extensions/pull/1946/commits/2cb8133579f0f6a36f78f4a32ef2a5287d1008e8